### ISSUE TITLE
Netflix Visualizations - Summary

### ISSUE THREAD

**Issue by jmcrey** [salleynealt/codecademy-dvp-2#7]
## Rubric Score
 
### Criteria 1: Pandas and CSV
* _Score Level:_ **4 (Exceeds Expectations)**
* _Comment(s):_
 
The Jupyter notebook imports the necessary libraries and converts the CSV to a Pandas DataFrame. 

### Criteria 2: Implementation of visualizations in spec
* _Score Level:_ **4 (Exceeds Expectations)**
* _Extra Credit_: 2 (Half a point for each of the extra graphs)
* _Comment(s):_
 
The Jupyter notebook renders all Seaborn and Matplotlib visualizations and content provided in the design specification. These graphs were _incredible_. I mean, truly on a different level. Especially the custom charts -- they added so much information to your analysis and gave the project a solid backbone. Everything from the title, to the colors, to the labeling of the axes added value to these graphs. Moreover, the actual implementation of them was very clever and utilized a lot of built-in Python functionality. I could tell that a lot of these graphs took dedication and time to figure out and craft. For this dedication, and the 4 extra graphs, I am awarding this section 2 extra credit points. Excellent job!

### Criteria 3: Color
* _Score Level:_ **4 (Exceeds Expectations)**
* _Comment(s):_
 
The color of each of these charts not only helped depict the essentials of the graph (as previously mentioned), they meshed well with the overall theme of the presentation. It seems a lot of time went into choosing these colors to help match the presentation theme. A personal favorite was using RGB colors to choose a custom shade. Great job!

### Criteria 4: Style
* _Score Level:_ **4 (Exceeds Expectations)**
* _Comment(s):_

The chart labels and titles are concise, easy to read, and they give a great description of what the viewer is looking at. Moreover, the analysis following the graphs are clear, easy to follow, and they give the reader the exact information they need to read the graphs. Plus, I love how succinct the bullet points are -- very to the point, which is great for a presentation. Nice job!

### Criteria 5: Presentation and framing of visualization
* _Score Level:_ **4 (Exceeds Expectations)**
* _Comment(s):_

Presentation cohesively presents every chart and has an appropriate description for each, with further research outlined in the notes. Plus, the 4 extra graphs gives way more insight into Netflix's overall financial status. The overall theme of the presentation was maintained throughout, and the message given was clear. A personal favorite was the conclusion because of how much information was depicted in a such few sentences. 

### Overall Score: **22/20**

The research and dedication that went into this project was apparent and very much appreciated. It made this project shine. The one point of improvement is formatting of the code to conform to Python's best practices. A lot of this is simply spacing, which I covered in [this issue](https://github.com/salleynealt/codecademy-dvp-2/issues/4). But, even then, you made great use of Python's built-in functions, which shows a deeper understanding of Python as a whole.

So, in summary, this project was a pleasure to read and displayed a great amount of tenacity and dedication. Excellent job!

**Comment by salleynealt** [comment]
Thank you very much https://github.com/jmcrey. I did put a lot of time and effort into the presentation. I will take the spacing and best practices into my future python projects and work.
